lib/concentrate/vehicle_position.ex

```diff
@@ -4,6 +4,8 @@ defmodule Concentrate.VehiclePosition do
   """
   import Concentrate.StructHelpers
 
+  alias Concentrate.{DataDiscrepancy, VehiclePosition}
```

Comment:
This file is aliasing itself? Can't it just use `.source` instead of `&VehiclePosition.source/1`? Other callers use functions to access the fields to keep the data type opaque, but in the same file, that's not a concern. If it does need to be opaque, can it use `&source/1` directly, without `VehiclePosition.`?

Reply:
Updated the source calls but this is still needed for the attribute accessors in the discrepancies function.